.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
sorry, a minor change, can you add the following

```
  labels:
    - "area/dependency"
    - "release-note-none"
    - "ok-to-test"
```

/lgtm cancel

Reply:
added the labels!

Reply:
thanks!